website/_data/events.yml

```
@@ -110,4 +110,29 @@
     - rachael_wright_munn
   guests:
   talks:
+- dot_seven:
+  name: v0.7
+  date: June 24th, 2021 19:00 UTC
+  description: RubyCentral, RailsGirls, and mentorship.
+  event_text: Ruby Galaxy July 🌌
+  hosts:
+    - rachael_wright_munn
+    - kirk_haines
+  guests:
+    - megan_tiu
+    - ramon_huidobro
+  talks:
+    - name: Ruby Central, Apprenticeships, and Ruby
+      abstract: Megan Tiu talks about RubyCentral, Apprenticeships, Ruby, and her work with RailsGirls
```

```suggestion
      abstract: Megan Tiu talks about RubyCentral, Apprenticeships, Ruby, and their work with RailsGirls
```

Megan reached out to me this morning and asked me to use they/them this stream.